index.js

```diff
@@ -65,8 +64,43 @@ function MLLPServer(host, port) {
 
     });
 
-    Server.listen(PORT, HOST);
+    self.send = function (receivingHost, receivingPort, hl7Data, callback) {
+        var sendingClient = new net.connect({
+            host: receivingHost,
+            port: receivingPort
+        }, function () {
+            console.log('Sending data to ' + receivingHost + ':' + receivingPort);
+            sendingClient.write(VT + hl7Data + FS + CR);
+        });
+
+        sendingClient.on('data', function (rawAckData) {
+            console.log(receivingHost + ':' + receivingPort + ' ACKED data');
+
+            var ackData = rawAckData
+                .toString() // Buffer -> String
+                .replace(VT, '')
+                .split('\r')[1] // Ack data
+                .replace(FS, '')
+                .replace(CR, '');
+
+            callback(null, ackData);
```

Comment:
@Eriatolc I'd like to get this PR in sooner than later when you have a moment. Please note the following items:

1) I've created a separate issue to implement the the remaining ACK types: https://github.com/amida-tech/mllp/issues/7 - this is an existing issue that can be dealt with or without this PR is merged in.

2) I didn't refactor the sender code to use promises. I went with the node callback "error first" style so at least it's relatively standard. I think folks will know what's going on.

Sound good?

Thanks,
-Matthew

cc: @jsachs